src/impl_owned_array.rs

```diff
@@ -774,7 +861,6 @@ where
     }
 }
 
-
 /// Sort axes to standard order, i.e Axis(0) has biggest stride and Axis(n - 1) least stride
 ///
 /// Axes in a and b are sorted by the strides of `a`, and `a`'s axes should have stride >= 0 before
```

Comment:
This PR should preferably have a test for the new method :slightly_smiling_face: